components/dashboard/src/components/DropDown.tsx

```diff
@@ -35,14 +36,20 @@ function DropDown(props: DropDownProps) {
             },
         };
     });
-    const font =
-        "text-gray-400 dark:text-gray-500 text-sm leading-1 group hover:text-gray-600 dark:hover:text-gray-400 transition ease-in-out";
+    const defaultFont = "text-gray-400 dark:text-gray-500 hover:text-gray-600 dark:hover:text-gray-400 ";
+    const asLinkFont = "text-blue-500 dark:text-blue-400 hover:text-blue-600 dark:hover:text-blue-500";
+    const asLinkArrowBorder =
+        "border-blue-500 dark:border-blue-400 group-hover:border-blue-600 dark:group-hover:border-blue-500";
     return (
-        <ContextMenu menuEntries={enhancedEntries} classes={`${props.contextMenuWidth} right-0`}>
-            <span className={`py-2 cursor-pointer ${font}`}>
+        <ContextMenu menuEntries={enhancedEntries} customClasses={`${props.customClasses} right-0`}>
```

Comment:
**thought**: Wondering if we need the custom classes altogether here. 💭

Reply:
I believe we do, because the context-menu sometimes need to be left-aligned or right-aligned, among other things.

Reply:
It's better to  never use right-aligned items inside the dropdown menus. 💡 

Do you have a handy instance of a left-alignment you're referring to?

I've only seen the dropdown in the workspace start page when using a desktop editor that's using center alignment and this should also change, see https://github.com/gitpod-io/gitpod/issues/6910.

If this is about the alignment of the component itself with the rest of the elements around it, we could probably move the alignment control outside this component, right?

Reply:
> It's better to only use left-aligned items inside the dropdown menus. 💡

Ah, sorry, I meant how the collapsible elements horizontally align with respect to the control, e.g.:

<table><tr><td>
<img width="409" alt="Screenshot 2022-05-30 at 16 01 13" src="https://user-images.githubusercontent.com/599268/171008352-719de8fa-9a28-4e63-a99c-9ce63383415d.png">
</td><td>
<img width="207" alt="Screenshot 2022-05-30 at 16 01 18" src="https://user-images.githubusercontent.com/599268/171008355-0bdbccc1-ab76-4027-8b22-91ee093c6d63.png">
</td><td>
<img width="373" alt="Screenshot 2022-05-30 at 16 01 31" src="https://user-images.githubusercontent.com/599268/171008356-7244eeae-c203-4861-859a-c51effa18975.png">
</td></tr><tr><td>
<img width="215" alt="Screenshot 2022-05-30 at 16 01 56" src="https://user-images.githubusercontent.com/599268/171008360-101db20d-151a-452b-9090-bd222278a403.png">
</td><td>
<img width="185" alt="Screenshot 2022-05-30 at 16 02 41" src="https://user-images.githubusercontent.com/599268/171008362-7a9e5ee7-20bf-4ed9-8bbc-d99938baef0f.png">
</td></tr></table>

Reply:
Ah, thanks for clarifying, @jankeromnes!